README.md

```diff
@@ -1,17 +1,29 @@
 # Morpho Token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Transferability and roles
+
+The Morpho token no longer has role-based permissions.
+This implies notably that it is transferable by default.
+
+### Burning tokens
+
+Calling `transfer` or `transferFrom` to transfer tokens to the zero address now reverts in the Morpho token.
+Instead, any account approved to spend tokens can use the `burn` function.
```

Comment:
This is unclear, I think that we should state as follows. Recall how burning tokens was supported on legacy. Say the new Morpho token brings braking changes. State how burning token is supported in the new version.

Reply:
```suggestion
This version bring a breaking change for this feature.
In the legacy version the burning tokens was supported by transferring the desired amount to burn to address zero.
This approach is deprecated in this version of the Morpho token.
To burn tokens, approved users may call the `burn` function.
```